app/src/main/java/com/example/myapplication/adapter/UsersAdapter.kt

```diff
@@ -1,35 +1,79 @@
 package com.example.myapplication.adapter
 
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
+import com.example.myapplication.R
 import com.example.myapplication.databinding.UserPatternBinding
 import com.example.myapplication.loadImage
 import com.example.myapplication.model.User
 
+private const val DEFAULT_MARGIN = 8
+private const val MULTISELECT_MODE_MARGIN = 40
 
 class UsersAdapter(private val actionListener: UserActionListener) :
     ListAdapter<User, UsersAdapter.UsersViewHolder>(MyItemCallback()) {
 
+    private val _isModeActive = MutableLiveData(false)
+    private val isModeActive: LiveData<Boolean> get() = _isModeActive
```

Comment:
I don't know if using livedata to handle multi-select state is a good idea. Ask at the mentor review about this in more detail.